index.bs

```diff
@@ -0,0 +1,505 @@
+  :: Returns true iff |handle| is a {{FileSystemDirectoryHandle}}.
+
+  : |handle| . {{FileSystemHandle/name}}
+  :: Returns the [=entry/name=] of the entry represented by |handle|.
+</dl></div>
+
+The <dfn attribute for=FileSystemHandle>isFile</dfn> attribute must return true if the associated
+[=FileSystemHandle/entry=] is a [=file entry=], and false otherwise.
+
+The <dfn attribute for=FileSystemHandle>isDirectory</dfn> attribute must return true if the
+associated [=FileSystemHandle/entry=] is a [=directory entry=], and false otherwise.
+
+The <dfn attribute for=FileSystemHandle>name</dfn> attribute must return the [=entry/name=] of the
+associated [=FileSystemHandle/entry=].
+
+### The {{FileSystemHandle/getParent()}} method ### {#api-filesystemhandle-getparent}
+
+<div class=note><dl class=domintro>
+  : |parent| = await |handle| . {{FileSystemHandle/getParent()}}
+  :: Returns a promise resolved to the [=parent directory=] of |handle|.
+     If |handle| is the [=root directory=] of its file system the promise
+     resolves to `null`.
+
+     Can reject if the entry no longer exists in the underlying file system, or
+     if some error prevents reading the entry or its parent.
+</dl></div>
+
+<div algorithm>
+The <dfn method for=FileSystemHandle>getParent()</dfn> method, when invoked, must run these steps:
+
+1. TODO
+
+</div>
+
+### The {{FileSystemHandle/moveTo()}} method ### {#api-filesystemhandle-moveto}
+
+<div class=note><dl class=domintro>
+  : |newHandle| = await |handle| . {{FileSystemHandle/moveTo()|moveTo}}(await
+        |handle|.{{FileSystemHandle/getParent()}}, |newName|)
+  :: Renames the entry represented by |handle| to |newName|.
+
+  : |newHandle| = await |handle| . {{FileSystemHandle/moveTo()|moveTo}}(|otherDir|)
+  :: Moves the entry represented by |handle| to |otherDir|, while keeping its existing name.
+
+  : |newHandle| = await |handle| . {{FileSystemHandle/moveTo()|moveTo}}(|otherDir|, |newName|)
+  :: Moves the entry represented by |handle| to |otherDir|, while renaming it to |newName|.
+
+     In all of these cases, |handle| will no longer represent a valid entry, and thus any further
+     operations on it will fail.
+
+     Attempting to move an entry to the directory it already exists in, without renaming it is
+     an error. In all other cases if the target entry already exists, it is overwritten.
+</div></dl>
+
+<div algorithm>
+The <dfn method for=FileSystemHandle>moveTo(|parent|, |name|)</dfn> method, when invoked, must run
+these steps:
+
+1. TODO
+
+</div>
+
+### The {{FileSystemHandle/copyTo()}} method ### {#api-filesystemhandle-copyto}
+
+<div class=note><dl class=domintro>
+  : |newHandle| = await |handle| . {{FileSystemHandle/moveTo()|copyTo}}(await
+        |handle|.{{FileSystemHandle/getParent()}}, |newName|)
+  :: Creates a copy of the entry represented by |handle| with name |newName|.
+
+  : |newHandle| = await |handle| . {{FileSystemHandle/moveTo()|moveTo}}(|otherDir|)

```

Comment:
nit: s/moveTo/copyTo/ here and in the next dt

Reply:
Done